MapboxNavigation/CarPlayManager.swift

```diff
@@ -141,9 +141,7 @@ public protocol CarPlayManagerDelegate {
 /**
  `CarPlayManager` is the main object responsible for orchestrating interactions with a Mapbox map on CarPlay.
  
- You do not create instances of this object yourself; instead, use the `CarPlayManager.shared` class property.
- 
- Messages declared in the `CPApplicationDelegate` protocol should be sent to this object in the containing application's application delegate. Implement `CarPlayManagerDelegate` in the containing application and assign an instance to the `delegate` property of the `CarPlayManager` shared instance.
+ Messages declared in the `CPApplicationDelegate` protocol should be sent to this object in the containing application's application delegate. Implement `CarPlayManagerDelegate` in the containing application and assign an instance to the `delegate` property of your `CarPlayManager` instance.
```

Comment:
We should probably note that the application should only have a single CarPlayManager instance at any given time. Effectively a singleton, but managed by the application instead of the SDK, to get around the access token problem.